hugo/content/docs/user_interaction.md

```diff
@@ -0,0 +1,111 @@
+---
+title: 'User Interaction'
+Weight: 900
+---
+
+## Interacting with a Sprotty Diagram
+Sprotty offers multiple mouse and keyboard interactions by default listed in the following tables.
+The `CTRL` key in the following is equal to the `CMD` key on Mac.
+
+| Mouse       
+| ----------------- | ---- |
+| left-click:       | toggle selection |
+| CTRL-click:       | add to/remove from selection |
+| left-click : drag | move selection (when on selected element) |
+| left-click : drag | pan viewport (when on diagram background) |
+| mouse wheel:    | zoom |
+
+| Touchpad
+| --------------------- | --- |
+|click                  | toggle selection |
+|drag                   | move selection (when on selected element) |
+|two-finger pan up/down | zoom |
+|zoom                   | zoom |
+
+| Keys
+| ---------- | --- |
+|CTRL-SHIFT-A| select all |
+|CTRL-SHIFT-C| center selection, or if nothing is selected center the entire diagram |
+|CTRL-SHIFT-E| export diagram to SVG |
+|CTRL-SHIFT-F| zoom selection to fill the entire canvas, or if nothing is selected zoom the entire diagram |
+|CTRL-Z      | undo |
+|CTRL-SHIFT-Z| redo |
+
+## Creating Custom Interaction
+
+### Buttons and Button Handlers
+The easiest way to create custom interactions is through buttons. Buttons in Sprotty work similarly to the other model elements. 
```

Comment:
```suggestion
Buttons in Sprotty work similarly to the other model elements, but they enable you to directly hook up handlers that react to the push of a button.
```

Not sure if we should claim implementing buttons are easier than mouse or keyboard listeners. :-)